pkg/tfvars/azure/azure.go

```diff
@@ -49,6 +50,7 @@ type TFVarsSources struct {
 	Auth                        Auth
 	CloudName                   azure.CloudEnvironment
 	ARMEndpoint                 string
+	RHCOSImage                  string
```

Comment:
I'm happy with the field name `RHCOSImage` but let's consider others for the sake of brainstorming. 

This field refers to the URL of a storage blob which contains a VHD with RHCOS. Personally I'm leaning to the name `VHDURL`

Other possibilities: `OSImageURL`, `VHDBlob`. 

To me `VHD

Reply:
Yeah I was also unsure about the field name, I'm happy for whatever and dont really have strong feelings one way or another

Reply:
Let's go with `VHDURL`